lib/src/base-styles/_nx-page-header.scss

```diff
@@ -39,61 +40,41 @@ $main-header-product-version-row-height: 1fr;
 .nx-product {
   align-self: center;
   box-sizing: border-box;
-  color: $nx-off-white;
-  display: -ms-grid;
-  display: grid;
-  grid-column-gap: $nx-spacing-xxs;
-  -ms-grid-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  grid-template-rows: $main-header-product-title-row-height $main-header-product-version-row-height;
-  -ms-grid-columns: $main-header-product-height $nx-spacing-xxs auto;
-  grid-template-columns: $main-header-product-height auto;
-  grid-template-areas:
-    "logo title"
-    "logo version";
+  color: $main-header-color;
+  display: flex;
   height: $main-header-product-height;
   text-decoration: none;
   min-width: $nx-sidebar-width;
 }
 
-.nx-product__logo {
-  -ms-grid-row: 1;
-  -ms-grid-row-span: 2;
-  -ms-grid-column: 1;
-  -ms-grid-row-align: center;
+.nx-product__branding {
   align-self: center;
-  box-sizing: border-box;
-  grid-area: logo;
-  height: 100%;
-  width: 100%;
 }
 
-.nx-product__name {
-  -ms-grid-row: 1;
-  -ms-grid-column: 3;
-  @include semi-bold();
+.nx-product__wordmark {
+  height: 21px;
+  margin-top: 13px; // fix vertical centering
+}
 
-  -ms-grid-row-align: center;
+.nx-product__vertical-divider {
+  align-self: center;
```

Comment:
Rather than separately specifying `align-self` on the flex items you could specify `align-items` on the parent

Reply:
> Rather than separately specifying `align-self` on the flex items you could specify `align-items` on the parent

I did that here: https://github.com/sonatype/sonatype-react-shared-components/pull/30/commits/b5967e477de0d203e96ecf12ddb8baeb42123e1b